compiler-api/src/main/java/com/squareup/anvil/compiler/api/CodeGenerator.kt

```diff
@@ -39,6 +51,14 @@ public interface CodeGenerator : AnvilApplicabilityChecker {
     module: ModuleDescriptor,
     projectFiles: Collection<KtFile>,
   ): Collection<FileWithContent>
+
+  public companion object {
+    /**
+     * The default [group][CodeGenerator.group] for code generators.
+     * A lower group value will be executed before any higher group value.
+     */
+    public const val GROUP_DEFAULT: Int = 0
```

Comment:
Could use an enum for this, too! Integers for priority levels tend to lose information fairly readily. An enum would retain ordering guarantees, but enforce naming and make it a lot easier to nav through and find out where a particular CodeGenerator sits in the group.